### PR TITLE
Remove activity summary section from profile screen

### PR DESCRIPTION
**Pull request by lucyxubroad**
# Summary

Remove activity summary section from profile screen and all related files

# Detailed Changes

- Remove activity summary section from profile screen and all related files
- Link profile screen to list screen

# Demonstration

Screen Shot 2020-06-20 at 6 51 37 PM

### CODE REVIEW

File: Flick/Cells/ListTableViewCell.swift [hunk 1]
```diff
@@ -86,6 +86,9 @@ class ListTableViewCell: UITableViewCell {
 
     @objc private func seeAllMedia() {
         // TODO: Implement see all media action
+//        let listViewController = ListViewController()
+//        self.navigationController?.pushViewController(listViewController, animated: true)
```
Comment (haiyingweng) [inline Flick/Cells/ListTableViewCell.swift:90]: is it purposely commented out?

File: Flick/Views/ModalView.swift [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+class ModalView: UIView {
+
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
```
Comment (haiyingweng) [inline Flick/Views/ModalView.swift:19]: what's this file for?

Comment (lucyxubroad) [reply Flick/Views/ModalView.swift:19]: its for my pop up haha.. its from a branch im working on right now.

File: Flick/Cells/ListTableViewCell.swift (hunk 1 above)
Comment (lucyxubroad) [reply Flick/Cells/ListTableViewCell.swift:90]: ya. I think later we're gonna want to go to lists from clicking the see all button

**Pull request merged**